demos/pyvista-terrain-following-mesh/app.py

```diff
@@ -1,9 +1,10 @@
 import dash
+from dash import dcc, html, Input, Output
```

Comment:
Nonblocking, but some styleguides/linters complain that this is a duplicate import. This is why most of our examples now explicitly import everything you're going to use - in this case that would add `Dash` and `no_update` which then lose the `dash.` prefix below - so you can drop the plain `import dash`.